doc/admin-guide/layer-4-routing.en.rst

```diff
@@ -49,21 +49,19 @@ SNI Routing
 ===========
 
 Currently the only directly supported layer 4 routing (as of version 8.0) is SNI based routing. This
-imposes some requirements on the traffic.
-
-*  The inbound connection must be TLS.
-*  The outbound destination must handle the HTTP ``CONNECT`` method.
+imposes the requirement on the traffic that the inbound connection must be TLS.
 
 SNI routing is configured by :file:`ssl_server_name.yaml`.
 
 If SNI Routing is enabled the initial "`CLIENT HELLO
 <https://tools.ietf.org/html/rfc5246#section-7.4.1.2>`__" data of an inbound TLS connection is
 examined to extract the "`SNI <https://tools.ietf.org/html/rfc3546#section-3.1>`__" value. This is
 matched against the configuration data to select an action for the inbound connection. In this case
-the option of interest is ``tunnel_route``. If this is set then |TS| will connect to the specified
-destination and issue an HTTP ``CONNECT`` request using the SNI value as the URL for the request.
-Because the destination and the ``CONNECT`` are the same in general it will be necessary to use
-a plugin to change the URL in the ``CONNECT``.
+the option of interest is ``tunnel_route``. If this is set then |TS| will TCP connect to the
```

Comment:
It would be more accurate to say that ATS synthesizes / converts the connection in to an HTTP `CONNECT` request. That is, the inbound connection is treated _as if_ the user agent had sent a `CONNECT` to the upstream.